contracts/container/contract.go

```diff
@@ -1199,7 +1197,7 @@ func getEACL(ctx storage.Context, cid []byte) ExtendedACL {
 func getContainer(ctx storage.Context, cid []byte) Container {
 	data := storage.Get(ctx, append([]byte{containerKeyPrefix}, cid...))
 	if data != nil {
-		return std.Deserialize(data.([]byte)).(Container)
+		return Container{Value: data.([]byte), Sig: interop.Signature{}, Pub: interop.PublicKey{}, Token: []byte{}}
```

Comment:
IIUC this will break deletion via old APIs, but OK.

Reply:
it will, yeah. I am forcing change